src/spikeinterface/generation/__init__.py

```diff
@@ -11,3 +11,8 @@
     generate_displacement_vector,
     generate_drifting_recording,
 )
+
+from ._template_database import (
```

Comment:
I would remove the "_" from the file name.

Reply:
OK, are you fine with the location of the file?

Reply:
The changed.